### PR TITLE
remove localmonero from merchants list

### PR DESCRIPTION
**Pull request by HardenedSteel**
https://localmonero.co/blog/announcements/winding-down

### CODE REVIEW

**Comment by nahuhh** [comment]
https://github.com/monero-project/monero-site/pull/2293

**Comment by HardenedSteel** [comment]
@nahuhh thanks, closing.

File: community/merchants/index.md [hunk 1]
```diff
@@ -85,7 +66,7 @@ meta_descr: merchants.descr
           </div>
         </div>
       </div>
-      <div class="right half no-pad-sm col-lg-6 col-md-6 col-sm-12 col-xs-12">
+      <div class="left half no-pad-sm col-lg-6 col-md-6 col-sm-12 col-xs-12">
```
Comment (plowsof) [inline community/merchants/index.md:69]: is this diff required?

Comment (HardenedSteel) [reply community/merchants/index.md:69]: yes the list needs to be reordered.

**Pull request closed without merging**